spec/cases/each_with_ar_mysql.rb

```diff
@@ -0,0 +1,13 @@
+require './spec/cases/helper'
+require "active_record"
+require "mysql2"
```

Comment:
might be easier to keep the mysql/pg/sqlite part in a case statement inside of each_with_ar_generic.rb